internal/querycoordv2/observers/collection_observer.go

```diff
@@ -140,23 +141,91 @@ func (ob *CollectionObserver) observeLoadStatus() {
 		if collection.LoadPercentage == 100 {
 			continue
 		}
-		ob.observeCollectionLoadStatus(collection)
+		// Explicitly ignoring return value.
+		_, _ = ob.observeCollectionLoadStatus(collection, false)
 	}
 
 	partitions := ob.meta.CollectionManager.GetAllPartitions()
 	if len(partitions) > 0 {
-		log.Info("observe partitions status", zap.Int("partitionNum", len(partitions)))
+		log.Info("observe partitions status", zap.Int("partitions count", len(partitions)))
```

Comment:
use camelCase to name the log fields, so that everyone can find the variable by the name when debug